x-pack/plugins/task_manager/server/queries/task_claiming.test.ts

```diff
@@ -982,7 +982,7 @@ if (doc['task.runAt'].size()!=0) {
           scope: ['reporting'],
           state: { baby: 'Henhen' },
           status: 'claiming',
-          taskType: 'foo',
+          taskType: 'yawn',
```

Comment:
someone needs more coffee! :-)

Reply:
Looks like Gidi was very creative in mock data names :)